### PR TITLE
Add phone to order shipping address model

### PR DESCRIPTION
**Pull request by FiveDigital**
You can enter a phone number for you shipping address but it won't be persisted since the column is missing in the model/database. 

| Questions        | Answers
| ---------------- | -------------------------------------------------------
| BC breaks?       | no
| Tests pass?      | yes
| Related tickets? | no
| How to test?     | Enter a phone number in your shipping address. After placing the order the phone number is saved in the `s_order_shippingaddress` table.

### CODE REVIEW

**Comment by mitelg** [comment]
hey @FiveDigital 
thanks for your PR. please have a look at the tests. some of them are failing due to your changes

**Comment by FiveDigital** [comment]
The CI says: 
> SQLSTATE[42S22]: Column not found: 1054 Unknown column 's11_.phone' in 'field list'

I have added the column in my migration. Perhaps the migration is not applied? Do I have to do more than just add the migration file?

**Comment by mitelg** [comment]
the latest migration number is 809. so you have to change it, to at least 810
otherwise the migration number is duplicated and will have no affect

**Comment by FiveDigital** [comment]
Okay that fixed the problem :) I don't know why I chose the wrong migration number. I thin my fork was'nt up to date when I created this PR.

**Comment by FiveDigital** [comment]
@mitelg All green now 👍

**Comment by mitelg** [comment]
hey @FiveDigital 
after talking with my colleagues, we think there are still missing some things. a new field must be nullable (migration and doctrine model) and the new field will not show up in the backend. after you have changed your PR we could schedule it.

**Comment by FiveDigital** [comment]
Done 👍

File: engine/Shopware/Models/Order/Shipping.php [hunk 1]
```diff
@@ -156,6 +156,13 @@ class Shipping extends ModelEntity
     private $city = '';
 
     /**
+     * Contains the phone number of the shipping address
+     * @var string $phone
+     * @ORM\Column(name="phone", type="string", length=40, nullable=false)
```
Comment (mitelg) [inline engine/Shopware/Models/Order/Shipping.php]: remove the nullable parameter

**Review by mitelg — CHANGES_REQUESTED** [review]

**Comment by FiveDigital** [comment]
Done 👍

**Review by mitelg — APPROVED** [review]

**Comment by mitelg** [comment]
oh, the tests are failing :see_no_evil:

**Comment by FiveDigital** [comment]
Ah new migration was added in the meantime so I have renamed my new one and rebased the PR.

**Comment by FiveDigital** [comment]
@mitelg Will this be merged in 5.2 or should I rebase/recreate this PR for 5.3?

**Comment by mitelg** [comment]
@FiveDigital I don't think we are going to merge this in 5.2. 
If you like, you can do a rebase. otherwise we will cherry pick your commit :wink:

**Comment by FiveDigital** [comment]
I have created a new PR for 5.3 https://github.com/shopware/shopware/pull/1172

**Pull request closed without merging**